scripts/lib/install

```diff
@@ -42,9 +42,9 @@ fi
 ## Options from environment variables.
 #
 # Specify options for apt.
-APT_OPTIONS="${APT_OPTIONS:-}"
+read -r -a APT_OPTIONS <<< "${APT_OPTIONS:-}"
```

Comment:
I'm not convinced this does the same as the old code; convince me?

Reply:
This converts `APT_OPTIONS` from a string to an array by word splitting.

```console
$ declare -p APT_OPTIONS
declare -- APT_OPTIONS="--foo --bar --baz"
$ read -r -a APT_OPTIONS <<< "${APT_OPTIONS:-}"
$ declare -p APT_OPTIONS
declare -a APT_OPTIONS=([0]="--foo" [1]="--bar" [2]="--baz")
```

(It’s important to make word splitting explicit so that ShellCheck can then automatically detect places where word splitting is accidental, e.g. because of missing double quotes.)